app/controllers/ask-for-more-time/ask-for-more-time.ts

```diff
@@ -28,18 +28,19 @@ const askForMoreTimeEvidenceUploadConfig: EvidenceUploadConfig = {
   cancelPath: paths.common.askForMoreTimeCancel,
   nextPath: paths.common.askForMoreTimeCheckAndSend,
   askForMoreTimeFeatureEnabled: false,
-  updateCcdEvent: Events.EDIT_TIME_EXTENSION,
+  updateCcdEvent: Events.MAKE_AN_APPLICATION.TIME_EXTENSION,
   addEvidenceToSessionFunction: function (evidences, req: Request) {
-    if (!req.session.appeal.askForMoreTime.evidence) {
-      req.session.appeal.askForMoreTime.evidence = [];
+    if (!req.session.appeal.makeAnApplicationEvidence) {
+      req.session.appeal.makeAnApplicationEvidence = [];
     }
-    req.session.appeal.askForMoreTime.evidence.push(evidences);
+    const applications: any = req.session.appeal.makeAnApplicationEvidence;
+    applications.evidence.push(evidences);
```

Comment:
applications.evidence does not exists, if you just do
`req.session.appeal.makeAnApplicationEvidence.push(evidences)` it should work